examples/shared_samples/enclosure.rb

```diff
@@ -40,6 +48,7 @@
 
 variant = OneviewSDK.const_get("API#{@client.api_version}").variant unless @client.api_version < 300
 
+# To run this example for API Verison 1600, you would need to mention API Versio 1200 or less for scope resource.
```

Comment:
not sure what exactly this comment means here.

Reply:
@soodpr Scope is not yet covered in API version 1600.

Reply:
@soodpr removed the comment for now, will check with Sowmya on considering Scope resource for API 1600.